src/tests/control/test_events.py

```diff
@@ -684,13 +684,12 @@ def test_delete(self):
         doc = self.get_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk)
         assert doc.select("button")
         doc = self.post_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk, {})
-        if self.event1.subevents.count() > 1:  # we created 2 subevents. Number of subevents left after one is deleted
-            assert doc.select(".alert-success")
+        assert doc.select(".alert-success")
         if self.event1.subevents.count() == 1:  # last one, can't be deleted
```

Comment:
The same as for the other if statement holds true here again :wink: This is never false, if everything goes well. Replace by ``assert self.event1.subevents.count() == 1`` :)

Reply:
yep, sorry man! I am new to unit tests, still trying to get a hold of it.